### ISSUE TITLE
multilabel models do not support custom eval_metric

### ISSUE THREAD

**Issue by TProrochenko** [catboost/catboost#1964]
Problem: multilabel models do not support custom eval_metric

```
import numpy as np
import catboost as cb

class CustomMultiError(object):
    def get_final_error(self, error, weight): 
        return error

    def is_max_optimal(self): 
        return False

    def evaluate(self, approxes, target, weight):
        error_sum = 1
        weight_sum = 1
        return error_sum, weight_sum
    
X = np.random.random(size = (100, 30))
y = np.random.random(size = (100, 5))

model = cb.CatBoostRegressor(
    loss_function='MultiRMSE',
    eval_metric = CustomMultiError(),
    iterations = 10, 
    learning_rate = 0.1
)

model.fit(X, y)
```

**Comment by LyzhinIvan** [comment]
Hi, @TProrochenko!

Sorry for very long response. The problem is easier than I think. For multi-target metrics it is required to inherit your class from `cb.MultiTargetCustomMetric`. For loss function it is required to inherit from `cb.MultiTargetCustomObjective`. We will add it to our tutorial.
